packages/terra-compact-interactive-list/src/orion-fusion-theme/Cell.module.scss

```diff
@@ -1,5 +1,11 @@
 :local {
   .orion-fusion-theme {
     --terra-compact-interactive-list-hover-background-color: #e0f2fb;
+    --terra-compact-interactive-list-cell-selected-background-color: #cbe7fa;
+
+    &.cell:focus {
+      --terra-compact-interactive-list-cell-focus-outline: none;
+      box-shadow: inset 0 0 1px 3px rgba(76, 178, 233, 0.5), inset 0 0 7px 4px rgba(76, 178, 233, 0.35);
```

Comment:
Should there be wdio tests to check for focus indicator?

Reply:
Yes, you are absolutely right. This PR grew to be much bigger than it was supposed to be and at this moment following wdio tests needs to be added:
1. Cell focus indicator.
2. Buttons and links inside of a cell focus indicator (should be their native focus styling). 
3. Selected cell background change.
I'll raise that question with the team, whether it should be done as a separate PR (as this one grew up to be a monstrosity and hard to maintain) or needs to be included in this PR.

Reply:
Here is a JIRA for tracking: UXPLATFORM-10113